evap/static/css/evap.scss

```diff
@@ -1037,6 +1037,28 @@ a.no-underline:hover {
     word-spacing: 30000px;
 }
 
+.separation-left {
+    border-left: 1px solid $medium-gray;
+}
+
+.flex-column {
+    flex-direction: column;
+}
+
+.clear-filter-button {
+    color: $darker-gray;
+    cursor: pointer;
+}
+.clear-filter-button:hover {
+    color: $dark-gray;
+}
+
+.result-filter-list {
+    max-height: 145px;
+    overflow-y: auto;
+    padding-left: 4px;
+}
```

Comment:
into its own file? at the very least i would add a comment that marks this a s the "results index" section